src/com/ean/clarity/Main.java

```diff
@@ -0,0 +1,53 @@
+package com.ean.clarity;
+
+import com.ean.clarity.util.Baugus;
```

Comment:
Good point.  We should move the contact malbera@ean.com line up to always fire instead of just on Baugus#fixProblem failure.